portal-web/test/functional/com/liferay/portalweb/macros/headless/objects/ObjectDefinitionAPI.macro

```diff
@@ -329,4 +348,19 @@ definition {
 		return "${getObjectsWithNestedFieldJSON}";
 	}
 
+	macro updateEmployee {
+		ObjectDefinitionAPI._updateEmployee(
+			employeeId = "${employeeId}",
+			firstname = "${employeeFirstname}",
+			newManagerId = "${newManagerId}");
+
```

Comment:
maybe it's safer to leave the name of the variable here as `managerId` instead of `newManagerId` as we may use this update method in the future for editing some other elements of the object.